go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/fsnotify/fsnotify
 
 go 1.16
 
-require golang.org/x/sys v0.0.0-20220908164124-27713097b956
+require (
+	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
```

Comment:
I'd rather not add a new dependencies, unless it really can't be avoided. Looking a bit at the source gocapability, we can probably just copy/paste some relevant bits in `internal/` or something. It's not a lot of code and mostly just a wrapper around `syscall`; it may be that the x/unix package solves this as well (didn't check yet).

Reply:
Agreed!I thought of removing them too. I'll change that.

Reply:
I was able to remove dependency on `syndtr/gocapability`.